### PR TITLE
test: unflake electron tests

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: tests/electron/globalSetup.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ import assert from 'assert';
 import { spawnAsync } from '../../packages/playwright-core/lib/utils/spawnAsync';
 
 export default async () => {
-  const result = await spawnAsync('npx', ['electron', require.resolve('./electron-print-chromium-version.js')], {
```
Comment (mxschmitt) [inline tests/electron/globalSetup.ts:21]: drive-by: in order to run the tests on a GitHub Codespace.

**Review by dgozman — APPROVED** [review]
